build.sbt

```diff
@@ -252,6 +253,7 @@ lazy val docs = (project in file("docs"))
       %%("fs2-io"),
       %%("fs2-cats"),
       %%("doobie-h2-cats"),
+      %%("play-test"),
```

Comment:
I could be mistaken, but I don't think we actually use _play-test_ in the docs as they are?